docs/dml-of-carbondata.md

```diff
@@ -402,6 +402,11 @@ CarbonData DML statements are documented here,which includes:
 
 ## UPDATE AND DELETE
 
+  Since the data stored in file system like HDFS is immutable, the update and delete in carbondata are done via maintaining two files namely:
```

Comment:
```suggestion
  Since the data stored in a file system like HDFS is immutable, the update and delete in carbondata are done via maintaining two files namely:
```

Reply:
Done